tests/test_client.py

```diff
@@ -96,3 +98,36 @@ def send_request(
         client.get_streams()
     assert err.value.http_code == 401
     assert http_client.counter == 1
+
+
+def test_get_jwt(client):
+    load_dotenv()
+    client._tenant.useFabric("_system")
+    user = client.create_user(
+        email="{}@macrometa.com".format(
+            "".join(random.choice(string.ascii_lowercase) for i in range(10))
```

Comment:
you don't need random join, import string and load_dotenv

you can get all the required info from the following:
`
    display_name = generate_username()
    email = "{}@macrometa.com".format(display_name)
    password = generate_string()
    user = client.create_user(email=email, password=password, display_name=display_name)
    username = user["username"]
    tenant = user["tenant"]
`